### DISCUSSION TITLE
Splash Page delivers empty <div>s instead of content

### DISCUSSION THREAD

**Discussion by foxler2010** [mmistakes/minimal-mistakes#3099] [Q&A] [answered]
## Environment

- Minimal Mistakes version: [mm-github-pages-starter commit](https://github.com/mmistakes/mm-github-pages-starter/commit/dfed5db462e438daac700832237b907834906e7d)
- Ruby gem or remote theme version: the one in mm-github-pages-starter
- Jekyll version: the (latest?) one used in GitHub pages
- Git repository URL: https://github.com/foxler2010/foxler2010.github.io
- GitHub Pages hosted (if yes provide URL to site): https://drewssite.top
- Operating system: Kubuntu 20.04/Firefox Browser

## Expected behavior

The homepage/first page of the site should be a splash page (as indicated in front matter) and contain the content that is also contained in the front matter.

## What Happened? How did this start?

When I created the page, it was copied from a template I found somewhere in the repositories for the theme, and I replaced (almost) all the content and removed some sections as well.

Later, as I was opening up my computer to work on stuff I browsed to the website and the whole content of the splash page was gone, except for the "Recent Posts" section I had added myself (it was not included in the template). Note that I added that section by copying a line of Liquid from somewhere else.

I checked to see if this was some sort of error in rendering the page, by using the "view-source:" tool that Firefox provides to see the source that was received by the browser. In all the spaces there was supposed to be content and splash page stuff, there were just empty \<div>s.

I have tried all sorts of stuff to fix this and it has done pretty much nothing to the end result. The content is always missing.

## Steps to reproduce the behavior

Go to https://drewssite.top and you will see some lines at the top of the page. This is where the splash content is supposed to be. It is not there currently.

## Other

What I see when I load the page:
![old_broken_splash](https://user-images.githubusercontent.com/49948988/128028848-0f357efd-ad98-445d-906a-b2153355ebc1.png)

there may be more posts there by the time i post this issue, i took the screenshot a while ago.

**Comment by iBug** [comment, marked as answer]
You have invalid YAML in your front matter:

<https://github.com/foxler2010/foxler2010.github.io/blob/master/index.md?plain=1#L48>
